servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -221,6 +212,9 @@ def _collect_relations(self, instance_config, offset):
         return self._get_json_batch(url, offset, instance_config.batch_size, instance_config.timeout, auth)
 
     def _process_relations(self, relations, instance_config):
+        """
```

Comment:
If we're adding this comment, let's add some more detail about what it does / returns

Reply:
I'll fix all pydoc when I'm finished with other refactoring